src/Tasks/Resources/Strings.resx

```diff
@@ -1338,6 +1338,10 @@
   <data name="RemoveDir.SkippingNonexistentDirectory">
     <value>Directory "{0}" doesn't exist. Skipping.</value>
   </data>
+  <data name="RemoveDir.EmptyPath">
+    <value>MSB3232: Detected an empty directory. Skipping.</value>
```

Comment:
Both you and @Nirmal4G had suggestions here - I like yours a bit better (sorry @Nirmal4G!) but I would change the name of the receiver from Delete to RemoveDir since that's the user-facing Task name that's being invoked.

```suggestion
    <value>MSB3232: An empty directory was passed to RemoveDir and was ignored.</value>
```

Reply:
No problem, I wasn't aiming to change the message too much!